src/Sylius/Bundle/AdminBundle/Resources/config/services.xml

```diff
@@ -28,6 +28,10 @@
     <services>
         <defaults public="true" />
 
+        <service id="Sylius\Bundle\AdminBundle\Command\CreateAdminUserCommand" autowire="true">
```

Comment:
In Sylius config, explicit arguments may be needed (to confirm, I just know for sure this is the case in Symfony core)

Reply:
which arguments ?

Reply:
@HeahDude I remove the `autowire="true"` and I added the service arguments in the xml